packages/react-ui-rent/.babelrc

```diff
@@ -1,45 +1,3 @@
 {
-  "env": {
-    "es": {
-      "presets": [
-        ["env", {
-          "modules": false,
-          "targets": {
-            "browsers": [">0.25%", "not op_mini all"]
-          }
-        }],
-        "react"
-      ],
-      "plugins": [
-        "transform-runtime",
-        "transform-decorators-legacy",
-        "transform-class-properties",
-        "babel-plugin-transform-object-rest-spread"
-      ]
-    },
-    "cjs": {
-      "presets": [
-        "env",
-        "react"
-      ],
-      "plugins": [
-        "transform-runtime",
-        "transform-decorators-legacy",
-        "transform-class-properties",
-        "babel-plugin-transform-object-rest-spread"
-      ]
-    },
-    "test": {
-      "presets": [
-        "env",
-        "react"
-      ],
-      "plugins": [
-        "transform-runtime",
-        "transform-decorators-legacy",
-        "transform-class-properties",
-        "babel-plugin-transform-object-rest-spread"
-      ]
-    }
-  }
+  "extends": "../../babel.config.js"
```

Comment:
👍

Reply:
There apparently is another way to do this using some `root` options for babel, but I couldn't get it work. This is just another way they say you can do it if you want.